contrib/example-config/jsonSchema.json

```diff
@@ -0,0 +1,14654 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$ref": "#/definitions/Welcome6",
```

Comment:
What is the meaning of the `Welcome` reference? Shouldn't it be named differently, e.g. `realmConfig`?